actions/notification_actions.jsx

```diff
@@ -166,6 +167,7 @@ const notifyMe = (title, body, channel, teamId, silent) => (dispatch, getState)
                     channel,
                     teamId,
                     silent,
+                    soundName,
```

Comment:
if the desktop app is less that `4.6.0` we shouldn't add the soundname property.

Reply:
Ok